projects.md

```diff
@@ -0,0 +1,10 @@
+# Projects
+
+Existing projects in Go that can be used as examples.
+
+## HelloFresh
+
+- https://github.com/hellofresh/remote-config-service
```

Comment:
This is a bit of the problem. How do we discover good internal project examples then?